### PR TITLE
Create chexslib.yml

### PR DESCRIPTION
**Pull request by softwaresale**
It took me a few tries, but I got my project file properly added....

### CODE REVIEW

**Comment by shiftkey** [comment]
@softwaresale just for reference, once you've opened a pull request you can continue to push commits to the branch, which will update the pull request. No need to close and create a new one!

**Comment by softwaresale** [comment]
@shiftkey Sorry about that. I was trying to create the file in browser to make life easier, but I got build errors and couldn't find the file.  Sorry about that

File: _data/projects/chexslib.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: chexslib
+desc: A
+site: https://github.com/softwaresale/chexslib
```
Comment (ritwik12) [inline _data/projects/chexslib.yml:3]: @softwaresale site is not available.

File: _data/projects/chexslib.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name: chexslib
+desc: A
```
Comment (ritwik12) [inline _data/projects/chexslib.yml:2]: @softwaresale Please give a brief description.

File: _data/projects/chexslib.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+site: https://github.com/softwaresale/chexslib
+tags:
+- C++
+- C
+- Http
+- Web
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/softwaresale/chexslib/issues/1
```
Comment (ritwik12) [inline _data/projects/chexslib.yml:11]: @softwaresale It should be a label link rather than issues link.

**Comment by shiftkey** [comment]
Closing this out due to inactivity.

@softwaresale please open a fresh PR if you would still like the project to be listed.

**Pull request closed without merging**